components/Peer/lib.ts

```diff
@@ -0,0 +1,3 @@
+import Button from "./Button";
+
+export { Button };
```

Comment:
What is this file's purpose? I'm not sure I understand its intention.

Reply:
its so that when more Peer styled components are made, they can all be imported from one file. see how in `StrollButton.tsx` I write `import * as Peer from ...` ? then I can write the component as `Peer.Button`, making it clear that it is not from react native.

Reply:
Gotcha! Thanks for explaining!